source/manage/server-admin-gui.rst

```diff
@@ -116,9 +116,11 @@ checks on a registered instance and reports the findings.
 
 |stt| is disabled by default.
 
-It can be enabled in :menuselection:`PMM Settings --> Settings --> Advanced Settings --> Security Threat Tool`.
+It can be enabled in :menuselection:`PMM --> PMM Settings --> Settings --> Advanced Settings --> Security Threat Tool`.
 
+The checks take 24 hours to complete.
```

Comment:
I think we should talk about how it works a little...it takes 24 hrs to complete seems ambiguous...when are checks downloaded?  24 hrs after enabled, immediately?  When are the scans performed?  24 hrs after enabled...every night at midnight...other?

Reply:
perhaps it makes sense to even say that all checks are downloaded from our central service too...

Reply:
yeah, I'll add additional PR to add more Text based on blogPost content  I have . 
 this one  more to clean up the page and have proper links from PMM Settings